GTG/gtk/application.py

```diff
@@ -121,19 +114,49 @@ def __init__(self, app_id, debug):
     def do_activate(self):
         """Callback when launched from the desktop."""
 
+        self.init_shared()
+
+        log.debug("Application activation finished")
+
+    def do_open(self, files, n_files, hint):
+        # Note: n_files isn't in the python doc, but is in the c doc and is required
```

Comment:
I'd rather leave this information in the commit. Nobody is calling this function directly and the docs might change